drgn_tools/bt.py

```diff
@@ -515,6 +516,20 @@ def bt_has_any(
         return _indexed_bt_has_any(prog, funcs)
 
     frame_list = []
+    if task is not None:
```

Comment:
nit: `task` should be documented in the docstring. Unless I find other issues though, I'll merge and fix it up myself.